src/molgenis_fdp_harvester/ckan_harvest/baseparser.py

```diff
@@ -0,0 +1,1158 @@
+# SPDX-FileCopyrightText: Open Knowlege
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# This material is copyright (c) Open Knowledge.
+# It is open and licensed under the GNU Affero General Public License (AGPL) v3.0
+# Original location of file: https://github.com/ckan/ckanext-dcat/blob/master/ckanext/dcat/profiles/base.py
+#
+# Modified by Stichting Health-RI to remove dependencies on CKAN
+
+import datetime
+import json
+from urllib.parse import quote
+import re
+
+from dateutil.parser import parse as parse_date
+from rdflib import term, URIRef, BNode, Literal
+from rdflib.namespace import Namespace, RDF, XSD, SKOS, RDFS
+from geomet import wkt, InvalidGeoJSONException
+
+# from ckantoolkit import config, url_for, asbool, get_action
+# from ckan.model.license import LicenseRegister
+# from ckan.lib.helpers import resource_formats
+# from ckanext.dcat.utils import DCAT_EXPOSE_SUBCATALOGS
+
+DCT = Namespace("http://purl.org/dc/terms/")
```

Comment:
A lot of the namespaces are also available within the rdf lib. Can be removed here